### PR TITLE
Mentioned TypeScript in the readme and made some stylistic changes

### PR DESCRIPTION
**Pull request by ehrencrona**
Mentioned the TypeScript conversion script in the README and made some stylistic changes to "Getting Started".

### CODE REVIEW

**Review by dummdidumm — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -1,44 +1,69 @@
 # sapper-template
 
-The default [Sapper](https://github.com/sveltejs/sapper) template, available for Rollup and webpack.
+The default template for setting up a [Sapper](https://github.com/sveltejs/sapper) project. Can use either either Rollup or webpack as bundler.
```
Comment (benmccann) [inline README.md:3]: typo: either either

File: README.md [hunk 2]
```diff
@@ -1,44 +1,69 @@
...
+
+ * Add `typescript` as well as typings as dependences in `package.json`
+ * Configure the bundler to use [`svelte-preprocess`](https://github.com/sveltejs/svelte-preprocess) and transpile the TypeScript code.
+ * Add a `tsconfig.json` file
+ * Update the project code to TypeScript
+
+The template comes with a script that will perform these changes for you by running
+
+```bash
+node scripts/setupTypeScriptRollup.js
```
Comment (benmccann) [inline README.md:61]: Conduitry suggested we drop `Rollup` from the name of this script and only create it for Rollup projects by modifying the scripts in https://github.com/sveltejs/sapper-template/tree/master/_template

Comment (ehrencrona) [reply README.md:61]: I saw that; I can create a new issue for it.

File: README.md (hunk 1 above)
Comment (ehrencrona) [reply README.md]: Thanks! Fixed.

**Comment by ehrencrona** [comment]
Could we merge this?

**Pull request merged**